README.md

```diff
@@ -96,9 +97,19 @@ python -m pip install .
 
 pip install "cbc.block[haznics] @ git+https://github.com/blocknics/cbc.block"
 ````
+(API)=
+# API
+The API of `CBC-block` can be found here:
+```{eval-rst}  
```

Comment:
I think we should just have a plain https link to docs, otherwise it looks bad on github,
![image](https://user-images.githubusercontent.com/266544/225260594-80278031-4d6c-4f6b-a54c-8f8fb8e7468c.png)